### PR TITLE
Make readme more user friendly

### PR DESCRIPTION
**Pull request by sdankel**
Making the readme more user-friendly and moving the testing related information to `docs/`

image

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,35 @@
-# Sway Visual Studio Code Plugin
+<picture>
+   <source media="(prefers-color-scheme: light)" srcset="./images/light/sway.png">
+   <source media="(prefers-color-scheme: dark)" srcset="./images/dark/sway.png">
+   <img alt="Sway VSCode Plugin" src="./images/dark/sway.png">
+</picture>
```
Comment (JoshuaBatty) [inline README.md:5]: nice!

File: README.md [hunk 2]
```diff
@@ -1,46 +1,35 @@
...
 
 [![Visual Studio Marketplace Version](https://img.shields.io/visual-studio-marketplace/v/FuelLabs.sway-vscode-plugin)](https://marketplace.visualstudio.com/items?itemName=FuelLabs.sway-vscode-plugin)
 [![discord](https://img.shields.io/badge/chat%20on-discord-orange?&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/xfpK4Pe)
 
-## When does the plugin get activated?
+This extension provides support for the Sway smart contract programming language.
```
Comment (JoshuaBatty) [inline README.md:10]: "This extension provides LSP support for the Sway smart contract programming language"

File: README.md [hunk 3]
```diff
@@ -1,46 +1,35 @@
...
 
-Currently it gets activated once you open a file with `.sw` extension.
+## Features
 
-## Testing
+- go to definition, implementation, type definition
```
Comment (JoshuaBatty) [inline README.md:14]: we don't currently support implementation or type definition, only go to definition. These can probably be removed for now.

File: README.md [hunk 4]
```diff
@@ -1,46 +1,35 @@
...
+- symbol renaming
```
Comment (JoshuaBatty) [inline README.md:15]: We don't yet support this capability either. We could add this to the coming soon section below though.

File: README.md [hunk 5]
```diff
@@ -1,46 +1,35 @@
-# Sway Visual Studio Code Plugin
+<picture>
+   <source media="(prefers-color-scheme: light)" srcset="./images/light/sway.png">
```
Comment (JoshuaBatty) [inline README.md:2]: let's wait until we get the updated images from Grant before landing this PR. They look really cool though!

File: README.md [hunk 6]
```diff
@@ -1,46 +1,35 @@
...
-## Testing as a real installed extension
+## Quick start
 
-```sh
-git clone git@github.com:FuelLabs/sway-vscode-plugin.git
-cd sway-vscode-plugin
-npm i
-npm run install-extension
-```
+1. Install the [Fuel toolchain](https://fuellabs.github.io/sway/latest/introduction/installation.html).
```
Comment (JoshuaBatty) [inline README.md:30]: I think pointing directly to the `fuelup` installation docs might be better here https://fuellabs.github.io/fuelup/master/installation/index.html

File: README.md [hunk 7]
```diff
@@ -1,46 +1,35 @@
...
+1. Install the [Sway VSCode plugin](https://marketplace.visualstudio.com/items?itemName=FuelLabs.sway-vscode-plugin).
```
Comment (JoshuaBatty) [inline README.md:31]: 2. Ensure `forc-lsp` is installed correctly by entering `forc-lsp --version`  into your terminal.
3. Install the ... (as above)

**Review by JoshuaBatty — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (sdankel) [reply README.md:2]: updated!

**Review by JoshuaBatty — APPROVED** [review]
:shipit:

**Pull request merged**